.travis.yml

```diff
@@ -22,6 +22,7 @@ gemfile:
   - gemfiles/rails-4-1-stable.gemfile
   - gemfiles/rails-4-2-stable.gemfile
   - gemfiles/rails-5-0-stable.gemfile
+  - gemfiles/rails-5-1-0-beta1.gemfile
```

Comment:
This is not to be added. We're already testing Rails master branch, where Rails 5.1.0 development is in progress.